aws_xray_sdk/core/models/entity.py

```diff
@@ -228,6 +230,20 @@ def add_exception(self, exception, stack, remote=False):
         self.cause['exceptions'] = exceptions
         self.cause['working_directory'] = os.getcwd()
 
+    def save_origin_trace_header(self, trace_header):
+        """
+        Temporarily store additional data fields in trace header
+        to the segment for later propagation. The data will be
```

Comment:
to the "Entity" for later propagation ...

Reply:
Done